### PR TITLE
[hack] Add JIRA status to errata script

### PR DESCRIPTION
**Pull request by sebsoto**
Adds a column to the errata build report indicating the status of the associated JIRA issue.

### CODE REVIEW

**Comment by sebsoto** [comment]
Example output:
```
Merge commits in release-4.14 after build windows-machine-config-operator-container-9.0.1-8
Commit Hash                              PR#  PR Title                                                          Jira ID       Fix Version   JIRA Status   Errata Status
======================================== ==== ================================================================= ============= ============= ============= =============
02ae878cff9bea85d10f38ab1641c326215e39d9 2009 [release-4.14] [controllers] Introduce err check when merging CA                                                         
ded42e9502853e90a7b4c7b4d210fa3cdf1a601e 1997 [release-4.14] Add support for Nutanix in machineset.sh                                                                  
934a89b81f5ad7ea5e0316e6d981dbb31a719018 2007 [release-4.14] OCPBUGS-27045,OCPBUGS-27046: Fix network config sc OCPBUGS-27045 4.14.z        MODIFIED      Not Attached
98a188acd644852df6c8a1f1724ab150ea644bc8 2010 [release-4.14] submodule update 01/12                                                                                    

Merge commits in release-4.14 since WMCO v9.0.0 included in build windows-machine-config-operator-container-9.0.1-8
Commit Hash                              PR#  PR Title                                                          Jira ID       Fix Version   JIRA Status   Errata Status
======================================== ==== ================================================================= ============= ============= ============= =============
086a61c9ab15f396099c8a3619102edfde79e4f8 1974 [release-4.14] OCPBUGS-24748: Ensure WICD service account token   OCPBUGS-24748 4.14.z        Verified      Attached
5430d02c10216e5fca1c84a6d995705f15a517f8 1992 [release-4.14] submodule update 01/02                                                                                    
37eef5847e73b5a06c71bd4ad75e33cd83f38d9f 1987 [release-4.14] submodule update 12 21                                                                                    
124be2f7121510a0ffa7598f44d52ca32aa076b7 1926 [release-4.14] OCPBUGS-22984: Introduce upgrading label to block  OCPBUGS-22984 4.14.z        Verified      Attached
6e82dfc4705a7dcbfc65e444b0c1800111325adc 1933 [release-4.14] [doc] Support cluster-wide proxy in README                                                                
642f47a685048455af5e0e210394b837ca9fc17c 1966 [release-4.14] Stop using automatically generated service account                                                        
ecbc335b467f2169046a3fc81032ef30fa426919 1964 [release-4.14] CSI documentation                                                                                         
a24ebc9c355797b0df1272855106d9f96abb4bf7 1960 [release-4.14] OWNERS: remove aravindhp                                                                                  
8ff22727335dd52c3b1cddff3d1728ed1ff01fe7 1950 [release-4.14] OCPBUGS-23286: Submodule update 11/16              OCPBUGS-23286 Restricted    Restricted    Attached
572db2e4295f8ed06829cd71e46653ad2f07954b 1938 [release-4.14] Add mansikulkarni96 as an approver                                                                        
f97167a0509265ec9c4399b1aee6ce485a5d6905 1963 [release-4.14] Update version to 9.0.1                                                                                   
```

**Comment by jrvaldes** [comment]
@sebsoto  consider overriding the e2e test suite here.

File: hack/errata-check.sh [hunk 1]
```diff
@@ -178,52 +178,75 @@ function get-jira-fix-version {
...
   else
-    fix_version=$(echo -E "$result" | jq -r '.fields.fixVersions[0].name')
+    fix_version=$(echo -E "$query_result" | jq -r '.fields.fixVersions[0].name')
     echo "$fix_version"
   fi
 }
 
+function get-jira-status {
+  local query_result=$1
+  local fix_version
```
Comment (mtnbikenc) [inline hack/errata-check.sh:207]:
```suggestion
  local status
```

**Review by mtnbikenc — APPROVED** [review]

**Comment by mtnbikenc** [comment]
Tested locally.
/lgtm

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator

**Pull request merged**